schema.sql

```diff
@@ -69,24 +75,24 @@ CREATE TABLE IF NOT EXISTS `OrderStatus` (
 CREATE TABLE IF NOT EXISTS `Review` (
     `customer_id` int  NOT NULL ,
     `product_id` int  NOT NULL ,
-    `rating` int  NULL ,
+    `rating` int  NOT NULL ,
     `review_text` nvarchar(255)  NULL ,
-    `review_date` datetime  NULL ,
+    `review_date` datetime  NOT NULL ,
     `review_header` nvarchar(255)  NULL ,
     PRIMARY KEY (
         `customer_id`,`product_id`
     )
 );
 
-CREATE TABLE IF NOT EXISTS `CustomerPassword` (
+CREATE TABLE IF NOT EXISTS `CustomerPasswords` (
     `customer_id` int  NOT NULL ,
     `hashed_password` binary(60)  NOT NULL ,
     PRIMARY KEY (
         `customer_id`
     )
 );
 
-CREATE TABLE IF NOT EXISTS `ManagerPassword` (
+CREATE TABLE IF NOT EXISTS `ManagerPasswords` (
```

Comment:
All table names should either be singular or plural, but we shouldn't mix them in the same DB.